NoSoft-SA/nspack
# Feature/reworks fruit actual counts

Summary of this change:
- expand unique constraint for fruit_actual_counts_for_packs
- change product setups' and reworks' actual_count field behaviours
- possible affected areas: carton labeling, reworks and palletizing (PDT and carton palletizing)

### Changed
- change fruit_actual_counts_for_packs table unique constraint
- change product setups' and reworks' actual_count field behaviours
- possible affected areas: carton labeling, reworks and palletizing (PDT and carton palletizing)

How to test this code via the UI:
* add fruit_actual_counts_for_packs options for same basic pack and std_fruit_size_count
Masterfiles | Fruit | Sizes | Std Fruit Size Counts | add actual count

*create and / or edit product setups - should have more options for fruit_actual_counts_for_packs
Production | Product Setups | Product Setup Templates | product setups

*edit pallet sequences - should have more options for fruit_actual_counts_for_packs
Production | Reworks | Single Pallet Edit | New
